Automattic-Tracks-iOS/Crash Logging/CrashLogging.swift

```diff
@@ -278,6 +286,104 @@ extension CrashLogging {
     }
 }
 
+// MARK: - Helpers for hybrid SDKs
+extension CrashLogging {
+    /// Returns the options required to initialize Sentry in other platforms.
+    public func getOptionsDict() -> [String: Any] {
+        return [
+            "dsn": self.dataProvider.sentryDSN,
+            "environment": self.dataProvider.buildType,
+            "releaseName": self.dataProvider.releaseName
+        ]
+    }
+
+    /// Return the current Sentry user.
+    /// This helper allows events triggered by other platforms to include the current user.
+    public func getSentryUserDict() -> [String: Any]? {
+        return dataProvider.currentUser?.sentryUser.serialize()
+    }
+
+    /// Attachs the current scope to an event and returns it.
+    /// This helper allows events triggered by other platforms to include the same scope as if they would be triggered in this platform.
+    ///
+    /// - Parameters:
+    ///   - eventDict: The event object
+    public func attachScopeToEvent(_ eventDict: [String: Any]) -> [String: Any] {
+        let scope = SentrySDK.currentHub().getScope().serialize()
+
+        // Setup tags
+        var tags = scope["tags"] as? [String: String] ?? [String: String]()
+        tags["locale"] = NSLocale.current.languageCode
+
+        /// Always provide a value in order to determine how often we're unable to retrieve application state
+        tags["app.state"] = ApplicationFacade().applicationState ?? "unknown"
+
+        tags["release"] = self.dataProvider.releaseName
+
+        // Assign scope to event
+        var eventWithScope = eventDict
+        eventWithScope["tags"] = tags
+        eventWithScope["breadcrumbs"] = scope["breadcrumbs"]
+        eventWithScope["contexts"] = scope["context"]
+
+        return eventWithScope
+    }
```

Comment:
The scope used in the events generated in hybrid SDKs is different from the one on the native side so I expose this function to incorporate the same data we see in native crashes.

This logic serializes the current scope and includes the same extra tags added in the `beforeSend` callback, finally, a new object is created with the following data set:
- Tags
- Breadcrumbs
- Contexts